tests/fixtures/rubric_simplified_obj.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
I think it is better to have all those rubric values in a json configuration file, and not check into github with real ids.

Reply:
OK, I will fake some values. I did not feel any harm using real values.

Reply:
Just to clarify I was extracting the Rubric JSON response and formatting and doing some clean up for easier user. From `Full "Dose"` or `No "Dose"` to "Full Dose/No Dose". I wanted clean Rubric object when assigning competencies. 

Initially when I coded for this the Rubric Criteria/Competencies value (Like Roles & Responsibilities) was different from Spreadsheet (Roles/Responsibilities) So I needed to do a data mapping. But after taking with IPE Leads team they agreed that Rubric Criteria/Competencies values should be identical to Gsheet. Hence the formatting around this Rubrics. 

I hope I make sense , Let me know if I am not clear

Reply:
@zqian I created this Rubric object just for unit tests, the real Prod will never read from this. Just to be clear.

Reply:
I changed the Rubric with dummy id. I Hope I address your concern checking in real id's Let me if know I y understanding is incorrect